### PR TITLE
Cleanup repo style

### PR DESCRIPTION
**Pull request by idanwe**
Following hapijs/contrib#31

* Move scripts to package.json and remove Makefile
* Change main file to lib/index.js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "lab": "5.x.x"
   },
   "scripts": {
-    "test": "make test-cov",
+    "test": "node node_modules/lab/bin/lab -a code -t 100 -L",
```
Comment (Marsup) [inline package.json]: NPM already has node_modules binaries in its path, simply call lab.

Comment (idanwe) [inline package.json]: I know it, I follow [hapi package.json](https://github.com/hapijs/hapi/blob/master/package.json#L53-L55)
Do you think that there is reason for the `node_modules` path there?

Comment (Marsup) [inline package.json]: I have no idea. @hueniverse ?

Comment (hueniverse) [inline package.json]: Copied from the Makefile... if it works without take it out (and open a PR on hapi)

Comment (Marsup) [inline package.json]: FWIWI this helps only on windows, because it doesn't understand shebangs, but I don't think we care.

Comment (nlf) [inline package.json]: IIRC npm installs a batch script that shims the bin to work in windows, so just pointing directly to lab should work just fine

Comment (Marsup) [inline package.json]: Yeah right, bad memory from trying to make lab work on windows the other day, I had to do that, my bad :)

**Pull request merged**